client/css/style.css

```diff
@@ -860,6 +860,10 @@ button,
 	color: #f00;
 }
 
+#chat .msg.toggle .time {
+	visibility: hidden;
+}
+
```

Comment:
I understand this commit is to keep a consistency on how messages are sent from the server to the client, in case some client changes / themes want to display the time for a link expander. Am I correct? Is there any other cases it would be useful?
I actually think it's fair to not even sent date from the server as there is essentially none.

Assuming I am correct, I think we should leave that out.

In the future, we might want to have a model for the link expander itself instead of deriving `Msg`, that would give us a bit more freedom I believe. But that's for another day.

Out of curiosity, why `visibility: hidden` instead of `display: none`? I think `display: none` would be better but I might be missing something.